deno.json

```diff
@@ -21,9 +21,7 @@
     "singleQuote": true
   },
   "lint": {
-    "files": {
-      "exclude": ["node_modules/", "test/", "vendor/", "dist/", "build/"]
-    }
+    "exclude": ["node_modules/", "test/", "scripts/binary.js", "test/assets/group.js", "test/assets/mailbox.js", "vendor/", "dist/", "build/"]
```

Comment:
Is it not possible to use glob expressions for things in `test/assets`? Also, why is `binary.js` being excluded? It doesn't look like it should be

Reply:
Got `NodeJS process global is discouraged in Deno` for `binary.js`

The script is classic Node CLI script so instead of ES modules refactor I thought it's best to ignore it.

Reply:
Also: the glob comment also applies to the other directories. It `test` didn't work before, the other directories also won't work